src/coreComponents/events/EventManager.cpp

```diff
@@ -224,39 +226,48 @@ bool EventManager::run( DomainPartition & domain )
 
 void EventManager::outputTime() const
 {
+  // The formating here is a work in progress.
+  GEOS_LOG_RANK_0( GEOS_FMT( "\n"
+                             "------------------- TIMESTEP START -------------------\n"
```

Comment:
```
------------------- TIMESTEP START -------------------
    - Time:       -3168y, -319d, -4h-1m-4s (-100000000000 s)
    - Delta Time: 3168y, 319d, 04h01m04s (100000000000 s)
    - Cycle:      0
------------------------------------------------------
```
@MelReyCG see the output above and could you please double check that conversion to h-m-s works correct?